### ISSUE TITLE
Prettier not formatting this line correctly

### ISSUE THREAD

**Issue by hesher** [prettier/prettier#2100]
Prettier can't format this line. (Line width = 110)

```
import ActiveSitesTableComponent, {ActiveSitesStateProps} from '../../components/SitesTable/ActiveSitesTableComponent';
```
The result is a line that is too long.

**Comment by k15a** [comment]
There is a similar issue with typescript type import.

**Comment by k15a** [comment]
The issue is already fixed in #2096
